src/Messaging/src/RabbitMQ/Steeltoe.Messaging.RabbitMQ.csproj

```diff
@@ -16,6 +16,7 @@
   <ItemGroup>
     <ProjectReference Include="..\..\..\Common\src\Common.Expression\Steeltoe.Common.Expression.csproj" />
     <ProjectReference Include="..\..\..\Common\src\Common.RetryPolly\Steeltoe.Common.RetryPolly.csproj" />
+    <ProjectReference Include="..\..\..\Connectors\src\ConnectorBase\Steeltoe.Connector.ConnectorBase.csproj" />
```

Comment:
I don't see this used anywhere

Reply:
Might have added while troubleshooting - let me remove and see.

Reply:
J/k it's required and used in RabbitServicesExtensions